cluster/saltbase/pillar/privilege.sls

```diff
@@ -1,2 +1,2 @@
 # If true, allow privileged containers to be created by API
-allow_privileged: false
+allow_privileged: true
```

Comment:
coreos/systemd and ansible doesn't have salt installed. That is why they have such flag hard-coded at the first place. I removed this flag from them since flag is set to be true by default now.

But on cloud providers which has salt, I leave the .sls file here so that the user can update this if they want to disable it. Also flag such as --allow-privileged has to have value associated with it. If it is empty, both kube-apiserver and kubelet will crash without startup.